samples/client/petstore/typescript-angular-v10-provided-in-root/builds/default/.openapi-generator/FILES

```diff
@@ -9,11 +9,11 @@ configuration.ts
 encoder.ts
 git_push.sh
 index.ts
-model/./apiResponse.ts
-model/./category.ts
-model/./order.ts
-model/./pet.ts
-model/./tag.ts
-model/./user.ts
+model/apiResponse.ts
+model/category.ts
+model/order.ts
+model/pet.ts
+model/tag.ts
+model/user.ts
```

Comment:
@beytun thanks for the PR. Just curious, are you using Git Bash on Windows to update the samples?

Reply:
Nope, I run the script in zsh on Mac.

Reply:
I see that "ci/circleci: build" failed, but don't know the issue. Can anyone guide me if I need to do anything?

Reply:
@beytun CircleCI generate code by it's own and try to match yours changes and his own changes. In the given file you have changed separator(from `./` to `/`) which doesn't match to what is generated in CircleCI

Reply:
Thanks, @borsch. After your comment, I tried other shells to generate code but the result is some. So I manually changed those files back to original state. Now CircleCI is also successful.